frontend/hub/remotes/RemoteForm.tsx

```diff
@@ -52,14 +61,15 @@ export function CreateRemote() {
         submitText={t('Create remote')}
         onSubmit={onSubmit}
         onCancel={() => navigate(-1)}
-        defaultValue={{ name: '', url: '' }}
+        defaultValue={{ name: '', url: '', requirements_file: yamlRequirementsTemplate }}
```

Comment:
this would prefill the requirements file with the sample, right?

Note that the requirements file is an optional value, mostly you'll want it emtpy, except when syncing from huge remote repositories.

Reply:
I changed the logic, if the user open the yaml editor the sample code is there as a helper, but it does not get sent to the backend if the user does not change that value.